### PR TITLE
Expose collection metadata

### PR DESCRIPTION
**Pull request by Ndpnt**

### CODE REVIEW

File: metadata.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+versions: https://github.com/OpenTermsArchive/demo-versions
+snapshots: https://github.com/OpenTermsArchive/demo-snapshots
+donations: https://opencollective.com/opentermsarchive
+logo: https://opentermsarchive.org/images/collections/demo.png
+languages: [en]
+jurisdictions: [EU]
+trackingPeriods:
+  startDate: 2023-04-23
+  schedule: "30 */12 * * *"
+  serverLocation: Roubaix, FR
```
Comment (Ndpnt) [inline metadata.yml:21]: Can you confirm @clementbiron ?

File: metadata.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      logo: https://opentermsarchive.org/images/logo/logo-open-terms-archive-black.png
+  sponsors: 
+    - name: Ministry for Europe and Foreign Affairs
+      url: https://www.diplomatie.gouv.fr/en/
+      logo: https://opentermsarchive.org/images/contributors/meae.png
+
+i18n:
+  fr: 
+    name: Démo
+    tagline: Services nécessaires au fonctionnement du moteur Open Terms Archive
```
Comment (clementbiron) [inline metadata.yml:45]:
```suggestion
    tagline: Sous-ensemble de services utilisés par Open Terms Archive
```

File: metadata.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    description: | 
+      La collection Demo suit les modifications apportées aux conditions d’utilisation d’un sous ensemble de services utilisés par Open Terms Archive.
+
+      Cela permet à l’équipe coeur d’ Open Terms Archive d’être elle même utilisatrice du logiciel qu’elle produit et de fournir une collection de référence.
```
Comment (clementbiron) [inline metadata.yml:49]:
```suggestion
    Cela permet à l’équipe coeur d’Open Terms Archive d’être elle même utilisatrice du logiciel qu’elle produit et de fournir une collection de référence.
```

File: metadata.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
+id: demo
+name: Demo
+tagline: Services needed to operate the Open Terms Archive engine
```
Comment (clementbiron) [inline metadata.yml:3]:
```suggestion
tagline: Subset of services used by Open Terms Archive
```

File: metadata.yml (hunk 1 above)
Comment (clementbiron) [reply metadata.yml:19]: Yes

**Review by clementbiron — CHANGES_REQUESTED** [review]

**Review by clementbiron — APPROVED** [review]

**Pull request merged**